patches/tModLoader/Terraria/ModLoader/VanillaDamageClasses.cs

```diff
@@ -4,26 +4,26 @@ namespace Terraria.ModLoader
 {
 	public class Melee : DamageClass
 	{
-		internal protected override string DisplayNameInternal => Language.GetTextValue("LegacyTooltip.2");
+		internal protected override string DisplayNameInternal => " "+Language.GetTextValue("LegacyTooltip.2");
```

Comment:
These locals already have spaces in them for some reason. It would actually make more sense to remove them here, to not force modders to add them.
![image](https://user-images.githubusercontent.com/6795251/99958956-4f62fc00-2d9b-11eb-84cf-922c398d033e.png)

Reply:
I totally misread the `$"{(classIsVanilla ? string.Empty : " ")` logic. That's exactly what the default logic does, gives modders a space. Lets restore that and instead do `.Substring(0)` inside the vanilla `Language.GetTextValue`

That is, assuming all languages have that translation start with a space